app/grandchallenge/evaluation/views/__init__.py

```diff
@@ -535,7 +535,36 @@ def get_context_data(self, *args, **kwargs):
         return context
 
 
-class EvaluationDetail(ObjectPermissionRequiredMixin, DetailView):
+class EvaluationIncompleteJobsMixin(ObjectPermissionRequiredMixin, DetailView):
+    model = Evaluation
```

Comment:
The Mixin can be leaner, otherwise looks good to go!

```suggestion
class EvaluationIncompleteJobsMixin:
```